src/components/Chat/Chat.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
-import { Button, Comment, Icon, Input, Segment } from 'semantic-ui-react';
+import {
+  Button,
+  Comment,
+  Icon,
+  Input,
+  InputOnChangeData,
+  Segment,
+} from 'semantic-ui-react';
 import { Socket } from 'socket.io';
 
 import { formatTimestamp, getColorHex, getDefaultPicture } from '../../utils';
+import { generateName } from '../../utils/generateName';
+import ChatVideoCard from '../ChatPlaylistCard';
 
 interface ChatProps {
+  userName: string;
```

Comment:
I'm not sure we should integrate name change into the Chat component. That would make it show up in theater mode, which I'm not sure is something we want.